### ISSUE TITLE
A11y: calcite-action-bar (Unable to override text spacing of calcite actions)

### ISSUE THREAD

**Issue by rslibed** [Esri/calcite-components#1149]
Reported from Level Access:

Module Name: 01 - Left Navigation

Issue: Spacing is unable to be overwritten on configuration panel buttons. As seen in screenshots below, when an alternative spacing option is applied, elements in the left hand panel do not update along with the remaining elements of the configuration panel and app.

Default Spacing: 
![06902a00-e225-11ea-83d4-1ea61b75adab](https://user-images.githubusercontent.com/29421380/96644026-4c2caa00-12dd-11eb-8942-8d6664187799.png)

New Style Applied:
![2e7f8d80-e225-11ea-8e20-b27be6e5242b](https://user-images.githubusercontent.com/29421380/96644185-8b5afb00-12dd-11eb-8429-b97b16226507.png)

[User Impact]
When text spacing cannot be overridden, low vision or dyslexic users may have difficulty reading the text.

[Code Reference]
<div class="text-container text-container--visible">Map</div>

[Recommendation]
Ensure that when users override text spacing via style sheet or another browser setting, there is no loss of content or functionality -- text must not be cut off or missing.
The content should adapt up to the following specific spacing requirements identified in WCAG 2.1:

- Line height must be able to adapt to 1.5 times the font size.
- Letter spacing must be able to adapt to 0.12 times the font size.
- Word spacing must be able to adapt to 0.16 times the font size.
- Spacing underneath paragraphs must be able to adapt to 2 times the font size.

Reference documentation, including information on how to test this:
https://labs.levelaccess.com/index.php/Text_Spacing_Favlet

**Comment by macandcheese** [comment]
Still valid, maybe a shadow dom limitation...

**Comment by julio8a** [comment]
valid
